android/src/main/java/com/applovin/reactnative/AppLovinMAXAdViewUiComponent.java

```diff
@@ -144,7 +145,9 @@ public void onAdLoaded(@NonNull final MaxAd ad)
     {
         WritableMap adInfo = AppLovinMAXModule.getInstance().getAdInfo( ad );
 
-        AppLovinMAXModule.getInstance().sendReactNativeEvent( "OnNativeUIComponentAdviewAdLoadedEvent", adInfo );
+        // Copy adInfo since it is consumed once it is sent
+        WritableMap adInfoForPreload = ( containerView != null ) ? adInfo.copy() : adInfo;
+        AppLovinMAXModule.getInstance().sendReactNativeEvent( "OnNativeUIComponentAdviewAdLoadedEvent", adInfoForPreload );
```

Comment:
`OnNativeUIComponentAdviewAdLoadedEvent` should actually be cased as `OnNativeUIComponentAdViewAdLoadedEvent`.